src/CoreWCF.Http/tests/ServiceContract/ITypedMessageTypedMethodMyService.cs

```diff
@@ -0,0 +1,235 @@
+    [ServiceContract()]
+    [XmlSerializerFormat(Style = OperationFormatStyle.Document, Use = OperationFormatUse.Literal)]
+    interface ITypedMessageTypedMethodMyService3
+    {
+        [OperationContract(IsOneWay = true, Action = "*")]
+        void MyOperation(FooMessage3 request);
+
+        [OperationContract]
+        int Method3(float num);
+    }
+
+    [MessageContract]
+    public class FooMessage3
+    {
+        [MessageBodyMember]
+        public int Name;
+
+        [MessageHeader(Name = "FooArray", Namespace = "http://FooNamespace")]
+        [XmlArray(ElementName = "FooArray", Namespace = "http://FooNamespace", IsNullable = true)]
+        [XmlArrayItem(ElementName = "FooA")]
+        public Foo[] foos;
+    }
+
+    [ServiceContract()]
+    [XmlSerializerFormat(Style = OperationFormatStyle.Document, Use = OperationFormatUse.Literal)]
+    interface ITypedMessageTypedMethodMyService4
+    {
+        [OperationContract(IsOneWay = true)]
+        void MyOperation(FooMessage4 request);
+
+        [OperationContract]
+        bool Method4(double dblnum, decimal decnum);
+    }
+
+    [MessageContract()]
+    public class FooMessage4
+    {
+        //        [MessageHeader(Name = "MyID")]
+        [MessageHeader(Name = "XmlID")]
+        [XmlElement(ElementName = "XmlID")]
+        public int ID;
+
+        //        [MessageHeader(Name = "MyNewID")]
+        [MessageHeader(Name = "XmlNewID")]
+        [XmlElement(ElementName = "XmlNewID")]
+        public int newID;
+
+        [MessageBodyMember(Name = "MyName")]
+        [XmlElement(ElementName = "XmlName")]
+        public string Name;
+
+        [MessageBodyMember(Name = "MyAddress")]
+        [XmlElement(ElementName = "XmlAddress")]
+        public string Address;
+    }
+
+    [ServiceContract()]
+    [XmlSerializerFormat(Style = OperationFormatStyle.Document, Use = OperationFormatUse.Literal)]
+    interface ITypedMessageTypedMethodMyService5
+    {
+        [OperationContract(IsOneWay = true, Action = "")]
+        void MyOperation(FooMessage5 request);
+
+        [OperationContract]
+        char Method5(string str, byte[] arrbyte);
+    }
+
+    [MessageContract]
+    public class FooMessage5
+    {
+        [MessageHeader]
+        public int Name;
+
+        [MessageBodyMember]
+        [XmlArray(ElementName = "FooArray", Namespace = "http://FooNamespace", IsNullable = true)]
+        [XmlArrayItem(ElementName = "FooA")]
+        public Foo[] foos;
+    }
+
+    [ServiceContract()]
+    [XmlSerializerFormat(Style = OperationFormatStyle.Document, Use = OperationFormatUse.Literal)]
+    interface ITypedMessageTypedMethodMyService6
+    {
+        [OperationContract(Action = "http://FooMessage6Action")]
+        void MyOperation(FooMessage6 request);
+
+        [OperationContract]
+        DateTime Method6(int num, Foo foo);
+    }
+
+    [MessageContract]
+    public class FooMessage6
+    {
+        // [MessageProperty(Name = "Property1")] MessageProperty don't support
```

Comment:
I'm not seeing any test code which would have exercised the MessageProperty attribute. The type of code I would expect to see to test this attribute would be some extensibility point being used, probably an IDispatchMessageInspector which checks that the message property has been added. Are these types reused in other tests which aren't included in this PR? Basically my concern is the MessageProperty attributes seem to be purposefully added but I'm not seeing any consequences of them being commented out. If they were never used in the original tests, just remove the attributes instead of commenting them out.

Reply:
I checked the original tests ,which never used, I will remove it